torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -945,6 +952,29 @@ def _is_packable_linear(match):
         weight_meta_value = linear_node.args[weight_idx].meta.get("val")
         if input_meta_value is None or weight_meta_value is None:
             return False
+        if linear_node.target == aten.bmm.default:
+            input_size = input_meta_value.size()
+            weight_size = weight_meta_value.size()
+            # bmm must be 3D tensor
+            assert len(input_size) == 3 and len(weight_size) == 3
+            if input_size[0] != weight_size[0]:
```

Comment:
If it is bmm, the first dim should always match? Should be assertion here?

Reply:
Thanks for the comment and changed.